sympy/printing/ccode.py

```diff
@@ -294,7 +321,107 @@ def indent_code(self, code):
         return pretty
 
 
-def ccode(expr, assign_to=None, **settings):
+class _C9XCodePrinter(object):
+    # Move these methods to C99CodePrinter when removing CCodePrinter
+    def _get_loop_opening_ending(self, indices):
+        open_lines = []
+        close_lines = []
+        loopstart = "for (int %(var)s=%(start)s; %(var)s<%(end)s; %(var)s++){"  # C99
```

Comment:
Is this the only incompatibility between the current CCodePrinter and C89Printer? This doesn't seem like much, not particularly worth making `ccode` always give a deprecation warning.